facts/versions.txt

```diff
@@ -7,7 +7,7 @@
 # starting with 8.10, all facts are generated by openvox packages or the facter gem release
 
 # Facter 4.11.0 - Released February 2025 and shipped with OpenVox 8.12.0
-8.12.1
+8.17.0
```

Comment:
I suggest updating the comment line too or people will get confused

Reply:
Facter version 4.11.0 did ship with Openvox 8.12.0 and all versions up through 8.17.0. What sort of information are we trying to communicate with the comment?